helmtestserver/server.go

```diff
@@ -17,16 +17,22 @@ limitations under the License.
 package helmtestserver
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
 
+	"golang.org/x/crypto/openpgp"
 	"helm.sh/helm/v3/pkg/action"
 	"helm.sh/helm/v3/pkg/repo"
 	"sigs.k8s.io/yaml"
 
 	"github.com/fluxcd/pkg/testserver"
 )
 
+const (
+	KeyRingName = "TestUser"
```

Comment:
I don't think this needs to be public.

Reply:
It's used twice, in `generateKeyring` and `packageChart`. Making it private 👍